conftest.py

```diff
@@ -46,6 +65,11 @@ def _get_storage_configs(request):
     }
 
 
+
+def _has_fixture(request, fixture):
+    return fixture in request.fixturenames
```

Comment:
Is there no way to simply check for the usage of the `benchmark` fixture from `pytest-benchmark`?

Reply:
that's what this is doing, i feel like it's pretty simple

Reply:
this was actually the simplest way i could find

Reply:
if you know of a better one, by all means let me know

Reply:
By any chance, are you re-implementing any one of the _already existing_ pytest-benchmark flags?
https://pytest-benchmark.readthedocs.io/en/stable/usage.html#commandline-options
There are a lot of them.

Reply:
no.

Reply:
Then I don't get it. How does this differ from `--benchmark-disable` or `--benchmark-skip` or `--benchmark-only`?

Reply:
a "full_benchmark" is a benchmark that takes a long time to run